File: APICuponSmart/src/java/modelo/pojo/respuesta/RespuestaEmpresa.java
package modelo.pojo.respuesta;

import java.util.List;
import modelo.pojo.entidad.Empresa;

public class RespuestaEmpresa extends Mensaje{
    private List<Empresa> contenido;
    
    public List<Empresa> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Empresa> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaEmpresa.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Empresa

class RespuestaEmpresa(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Empresa>? = null
): Mensaje(error, mensaje)